srcpkgs/gnome-games/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'gnome-games'
+pkgname=gnome-games
+version=3.30.2
+revision=1
+build_style=meson
+hostmakedepends="glib-devel pkg-config vala-devel"
+makedepends="libarchive-devel grilo-devel gtk+3-devel libglib-devel
+ libmanette-devel librsvg-devel libsoup-devel libxml2-devel retro-gtk-devel
+ sqlite-devel tracker-devel"
+short_desc="Browse and play your games"
+maintainer="Rasmus Thomsen <rasmus.thomsen@protonmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://wiki.gnome.org/Apps/Games"
+distfiles="${GNOME_SITE}/${pkgname}/${version%.*}/${pkgname}-${version}.tar.xz"
+checksum=5607c4071d3b07809ec1a88deb52bb289c29b2f5fb91374e244ebe7f8dee31a1
+nocross="depends on libmanette, which hard depends on gobject-introspection"
```

Comment:
Just leave without nocross since it's a package that it depends on and it will just fail without returning > 1.

Reply:
Ah, okay, wasn't sure about that.